README.md

```diff
@@ -1 +1,33 @@
-# census-rm-case-service
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/0c2913652fd94878b4c61838b54db11e)](https://www.codacy.com/app/zekizeki/rm-case-service?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ONSdigital/rm-case-service&amp;utm_campaign=Badge_Grade) [![Docker Pulls](https://img.shields.io/docker/pulls/sdcplatform/casesvc.svg)]()
+[![Build Status](https://travis-ci.org/ONSdigital/rm-case-service.svg?branch=master)](https://travis-ci.org/ONSdigital/rm-case-service)
+[![codecov](https://codecov.io/gh/ONSdigital/rm-case-service/branch/master/graph/badge.svg)](https://codecov.io/gh/ONSdigital/rm-case-service)
```

Comment:
Badges are all pointing at the old repo still

Reply:
fixed